pkg/crdutil/crdutil.go

```diff
@@ -49,20 +49,28 @@ func (s *StringList) Set(value string) error {
 }
 
 var (
-	crdsDir StringList
+	crdsDirs  StringList
+	crdsFiles StringList
 )
 
 func initFlags() {
-	flag.Var(&crdsDir, "crds-dir", "Path to the directory containing the CRD manifests")
+	flag.Var(&crdsDirs, "crds-dir", "Path to the directory containing the CRD manifests")
```

Comment:
given that this is crd specific util, we can just have one flag `-f` `--filename` which can be a file or directory. We can also use `-r` `--recursive` option to decide on recursively processing all yaml files in the directory which by default is false. This aligns with kubectl command as well.